mirgecom/boundary.py

```diff
@@ -666,7 +882,1095 @@ def isothermal_noslip_state(
     def temperature_bc(self, state_minus, **kwargs):
         r"""Get temperature value to weakly prescribe wall bc.
 
-        Returns $2*T_\text{wall} - T^-$ so that a central gradient flux
+        Returns $2T_\text{wall} - T^-$ so that a central gradient flux
         will get the correct $T_\text{wall}$ BC.
         """
         return 2*self._wall_temp - state_minus.temperature
```

Comment:
This might not be correct, or at least not consistent with (Mengaldo) [https://arc.aiaa.org/doi/10.2514/6.2014-2923], eqn. 50. and paragraph that follows.  I think the `temperature_bc` function used for the BC on `grad(T)` should send back `Tplus = Twall`, not `Tplus = 2Twall - Tminus` as it currently does.  It can result in negative `Tplus`.  Mengaldo's paper says to send `Twall*n` for the flux used in the grad(T) instead of using a numerical flux function, if I have the right read.